jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -116,8 +116,6 @@ def buildExclusions = [
 
   // Source / headers / docs -------------------------------
   [ /^osx1010-release-sources$/,      releaseType, gte(11) ],
-  [ /^osx1011-release-sources$/,      releaseType, lt(11)  ],
-  [ /^osx1011-release-sources$/,      releaseType, gte(12) ],
```

Comment:
Unless we've removed `osx1011-release-sources` from the job in the release CI, do we not need to still exclude `^osx1011-release-sources$` for Node.js 12+ to prevent two source jobs running on 12+ (one on `osx1011-release-sources` and one on `centos7-release-sources`)?

Reply:
@richardlau I can confirm that `osx1011-release-sources` is still a tick-able configuration in release build job (if that helps, I'm aware that you cannot access it to check.)

Reply:
@richardlau food-gun again, this one certainly does support flipping from exclude to include!

I've removed all the osx1011 labels (cc @AshCripps) from iojs+release and the pkg & tar labels for osx1010, leaving only osx1010-release-sources.

https://github.com/nodejs/jenkins-config-release/commit/93d3ad13a52a108dea7e37e1463848d1bf4b6405

```patch
-        <string>osx1010-release-pkg</string>
         <string>osx1010-release-sources</string>
-        <string>osx1010-release-tar</string>
-        <string>osx1011-release-pkg</string>
-        <string>osx1011-release-sources</string>
-        <string>osx1011-release-tar</string>
         <string>osx1015-release-pkg</string>
         <string>osx1015-release-tar</string>
```